lib/ansible/galaxy/api.py

```diff
@@ -33,8 +34,16 @@
 
 display = Display()
 _CACHE_LOCK = threading.Lock()
-COLLECTION_PAGE_SIZE = 100
-RETRY_HTTP_ERROR_CODES = [  # TODO: Allow user-configuration
+
+# multiplied with default page size to determine page size for operations with pagination
+# when one hasn't been specified for the server or with --{type}-page-size
+COLLECTION_PAGE_MULTIPLIER = 10
+ROLE_PAGE_MULTIPLIER = 5
+
+# Note: cloud.redhat.com masks rate limit errors with 403 (Forbidden) error codes.
+# Since 403 could reflect the actual problem (such as an expired token),
+# rate limit errors from AH cannot be retried by default.
+RETRY_HTTP_ERROR_CODES = [
```

Comment:
Future refactoring note: this could benefit from using `http.HTTPStatus` where possible.